### PR TITLE
feat: add platformadmin ut with tongyilingma

### PR DESCRIPTION
**Pull request by leossteven**
/kind feature

#### What this PR does / why we need it:
add platformadmin ut with tongyilingma

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

image

### CODE REVIEW

**Comment by leo952795** [comment]
hi @rambohe-ch , I'm participating in   [2024 Tianchi Cloud Native Programming Challenge - Track 3]
please review my PR, if there is anything I need to provide or modify, please let me know.

File: pkg/yurtmanager/webhook/platformadmin/v1alpha1/platformadmin_default_test.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
+package v1alpha1
```
Comment (rambohe-ch) [inline pkg/yurtmanager/webhook/platformadmin/v1alpha1/platformadmin_default_test.go:1]: this line should be moved the line 15 before `import`

File: pkg/yurtmanager/webhook/platformadmin/v1alpha2/platformadmin_default_test.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
+package v1alpha2
```
Comment (rambohe-ch) [inline pkg/yurtmanager/webhook/platformadmin/v1alpha2/platformadmin_default_test.go:1]: ditto

**Comment by rambohe-ch** [comment]
@leossteven please fix the comments for golangci lint errors.

**Comment by leossteven** [comment]
> @leossteven please fix the comments for golangci lint errors.

ok,  the imports has been adjusted according to the gci style.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**